tools/segmetrics/segmetrics.xml

```diff
@@ -314,6 +314,7 @@
         If all objects within your segmented images are *uniquely* labeled, you can turn on the "Segmentation is uniquely labeled" switch to speed up the computations. Leave it off otherwise, or if you are unsure. The same accounts for the "Ground truth is uniquely labeled" switch and your ground turth image data.
     </help>
     <citations>
+        <citation type="doi">10.5281/zenodo.10111958</citation>

```

Comment:
Is this really needed. Those citations are meant to help users to know - what to cite in a paper.

Galaxy can export citations for an entire analysis, but 5 citations seems a bit to excessive

Reply:
I agree that 5 citations are a bit excessive. The old 4 citations should be cited depending on which module of the tool has been used. The new one is the Zenodo entry of the tool, adding which was considered “good practice” I thought? Any idea how this can be solved more elegantly?

Reply:
I don't really know. My gut feeling is to leave it to proper publications.

Reply:
Alright! Lets trust your gut feeling :)